src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -335,13 +343,40 @@ Demos
 
 ## FAQ
```

Comment:
I shouldn't have said section. I should have said section if it's big enough or give some thought to where else it can be worked in. See starting up-related PR to your ebidel/headless2 for an example.

Your seeing lots of FAQs because I've only recently started pushing back.